test/test.js

```diff
@@ -111,4 +111,67 @@ new Reporter(testRunner, {
   projectFolder: utils.projectRoot(),
   showSlowTests: process.env.CI ? 5 : 0,
 });
+
+// Unconditionally generate testIDs for all tests and verify they don't clash.
+// This will add |test.testId| for every test.
+generateTestIDs(testRunner);
```

Comment:
nit: can we put this into its own file. I want to be able to easily remove the flakiness dashboard code if we decide its not worth it.

Reply:
Moved to `//test/utils.js` and consolidated in a single function call.